README.md

```diff
@@ -77,12 +89,41 @@ $ sudo zypper addrepo --refresh http://download.opensuse.org/repositories/devel:
 $ sudo zypper install slirp4netns
 ```
 
-#### [Debian GNU/Linux Sid](https://packages.debian.org/sid/slirp4netns)
+#### [Debian GNU/Linux (10 or later)](https://packages.debian.org/buster/slirp4netns) & [Ubuntu (19.04 or later)](https://packages.ubuntu.com/disco/slirp4netns)
 
 ```console
 $ sudo apt install slirp4netns
 ```
 
+You might need to enable user namespaces manually (not needed on Ubuntu):
```

Comment:
I believe this is an issue on Debian 9. In my testing, this is not an issue on Debian 10 (currently unreleased AKA "testing").